src/dao/MoodDao.java

```diff
@@ -45,6 +89,34 @@ public MoodDao() {
      * executing the query
      */
     public Mood getMoodById(int moodId) throws DaoException {
-        return null;
+        BufferedReader reader = null;
+        Mood rtnValue = null;
+        try {
+            String ln = "";
+            reader = new BufferedReader(new FileReader("data/Moods.csv"));
+            reader.readLine();
```

Comment:
Same as previous method. We only need this if the first line of the file is a header.